examples/gallery/maps/country_polygons.py

```diff
@@ -2,11 +2,12 @@
 Highlight country, continent and state polygons
 ===============================================
 
-The :meth:`pygmt.Figure.coast` method can highlight country polygons
-via the ``dcw`` parameter. It accepts the country code or full
-country name and can draw its borders and add a color to its landmass.
-It's also possible to define multiple countries at once by separating
-the individual names with commas.
+The :meth:`pygmt.Figure.coast` method can highlight country polygons via the ``dcw``
+parameter. It accepts the country code (following the 2-characters
```

Comment:
The wikipedia says "ISO 3166-1 alpha-2 codes are two-letter country codes defined in ISO 3166-1":
```suggestion
parameter. It accepts the country code (following the 2-letter
```

Reply:
Related PR for the GMT docs is at https://github.com/GenericMappingTools/gmt/pull/8763